packages/swap/prisma/schema.prisma

```diff
@@ -276,6 +276,18 @@ model Pool {
   @@schema("public")
 }
 
+model BoostPool {
+  id              Int           @id @default(autoincrement())
+  depositEnabled  Boolean
+  withdrawEnabled Boolean
+  boostEnabled    Boolean
```

Comment:
what will we use these for? usually we just read directly from RPC when required

Reply:
Hmm, well the RPCs will not return this info. Also, I think storing these details in the db is much more efficient as we will be able to change them based on events.
Additionally, we need them broken down in separate flags so we can determine exactly which part is disabled. This will give us more power when exposing boost details via the SDK. 

Hope this makes sense to you.